pkg/orchestration/wiring/platformdns/api/types.go

```diff
@@ -1,12 +1,12 @@
-package apiinternaldns
+package apiplatformdns
 
 import (
 	"github.com/atlassian/voyager"
 	"github.com/atlassian/voyager/pkg/servicecatalog"
 )
 
 const (
-	ResourceType                  voyager.ResourceType           = "InternalDNS"
+	ResourceType                  voyager.ResourceType           = "PlatformDNS"
```

Comment:
This is a breaking change. If we have any existing SDs with `InternalDNS` resource, they will break.
You may keep both the old and new resource type name for transition period until you migrate all SDs to a new name.

Reply:
Apparently there is no documentation for InternalDNS on DAC https://developer.atlassian.com/platform/micros2/reference/resources/
there should be one?

Reply:
Yes, this is intentionally a breaking change made now to avoid needing to care about compatibility, had we rolled out to users first. There are no resources outside the Micros teams (mostly personal test services, one belonging to a DE service in dev).